core/pkg/launch/job_builder.go

```diff
@@ -592,9 +592,12 @@ func (j *JobBuilder) Build(
 		if err != nil {
 			return nil, err
 		}
+		sourceInfo.InputTypes = data_types.ResolveTypes(map[string]interface{}{})
```

Comment:
Redundant {}?

Reply:
No, the second `{}` is being used to initialize an empty `map[string]interface{}`